<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ticket Problem</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/resource.qrc">
    <normaloff>:/Simple icons/simple_icons/fi-rr-print.svg</normaloff>:/Simple icons/simple_icons/fi-rr-print.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*
{
	color: rgb(79, 79, 79);
	background-color:  rgba(223, 223, 223, 200);
}


QPushButton
{ 
	font: 26pt &quot;MS Shell Dlg 2&quot;;
	alternate-background-color: rgb(255, 255, 255);
	
	background-color: rgb(185, 185, 185);
	border-width: 2px;
    border-radius: 10px;
	position: center;
 }

QPushButton:hover
{
	font: 26pt &quot;MS Shell Dlg 2&quot;;
	background: rgba(225, 225, 225, 180);
	border-style: dashed;
}

QPushButton:pressed
{
	font: 26pt &quot;MS Shell Dlg 2&quot;;
	background-color: rgb(136, 136, 136, 180);
	border-width: 2px;
    border-radius: 10px;
}

QLabel
{
	font: 26pt &quot;MS Shell Dlg 2&quot;;
}

QPlainTextEdit
{
	font: 10pt &quot;Cascadia Code&quot;;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="f_ticketProblem">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Retry</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
